nix/pkgs/mantis.nix

```diff
@@ -0,0 +1,89 @@
+{ src
+, lib
+, stdenv
+, makeWrapper
+, writeShellScriptBin
+, bash
+, protobuf
+, jdk
+, jre
+, gawk
+, sbt
+, impure ? false
+}:
+
+let
+
+  # sbt-protoc puts the scala plugin in /tmp/protobridge<some-random-number>.
+  # it is in fact a shell script with a standard `#!/usr/bin/env sh` shebang
+  # that makes the Nix sandbox ANGRY and breaks all the things in a cryptic,
+  # hairpull-inducing way. So we gotta sed it out. Not the prettiest thing
+  # but it works.
+  #
+  # This will be unnecessary in sbt-protoc>=1.0.0, as we can specify a file path
+  # Also, https://github.com/thesamet/sbt-protoc/issues/209 will need to be resolved
+  # for the plugins to not fail.
+  protoc-wrapper = writeShellScriptBin "protoc" ''
+    set -e
+
+    for f in "$@"; do
+      echo ''${f##*=}
+    done | grep protocbridge | xargs sed -i "1s|.*|#!${bash}/bin/bash|"
+
+    exec ${protobuf}/bin/protoc "$@"
+  '';
+  nativeBuildInputs = [ protoc-wrapper jdk makeWrapper ];
+
+  # read version from build.sbt
+  version = let
+    buildSbt = builtins.readFile ../../build.sbt;
+    captures = builtins.match ''.*version := "([^"]+)".*'' buildSbt;
+  in builtins.elemAt captures 0;
+
+  LD_LIBRARY_PATH = lib.makeLibraryPath [ stdenv.cc.cc.lib ];
+  PATH = lib.makeBinPath [ jre gawk ];
```

Comment:
Please add coreutils to `PATH`. It's required by the launcher scripts

Reply:
it's why I'm appending to the previous path, but I think your way is probably more correct, thanks :)

Reply:
fixed